Building DAG of jobs...
Full Traceback (most recent call last):
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/__init__.py", line 792, in snakemake
    success = workflow.execute(
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/workflow.py", line 895, in execute
    dag.init()
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 214, in init
    job = self.update([job], progress=progress, create_inventory=True)
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 927, in update
    raise exceptions[0]
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 885, in update
    self.update_(
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 1018, in update_
    raise ex
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 1000, in update_
    selected_job = self.update(
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 927, in update
    raise exceptions[0]
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 885, in update
    self.update_(
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 1018, in update_
    raise ex
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 1000, in update_
    selected_job = self.update(
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 927, in update
    raise exceptions[0]
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 885, in update
    self.update_(
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 1018, in update_
    raise ex
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 1000, in update_
    selected_job = self.update(
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 927, in update
    raise exceptions[0]
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 885, in update
    self.update_(
  File "/home1/zhuyixin/.conda/envs/assembly/lib/python3.10/site-packages/snakemake/dag.py", line 1046, in update_
    raise MissingInputException(job, missing_input)
snakemake.exceptions.MissingInputException: Missing input files for rule lociLocation:
    output: /home1/zhuyixin/zhuyixin_proj/AssmQuality/igGene/test.pri.txt, /home1/zhuyixin/zhuyixin_proj/AssmQuality/igGene/test.alt.txt
    wildcards: HOME=/home1/zhuyixin/zhuyixin_proj/AssmQuality, species=test
    affected files:
        NA

MissingInputException in rule lociLocation in file /home1/zhuyixin/ig-assembly-eval/Snakefile, line 85:
Missing input files for rule lociLocation:
    output: /home1/zhuyixin/zhuyixin_proj/AssmQuality/igGene/test.pri.txt, /home1/zhuyixin/zhuyixin_proj/AssmQuality/igGene/test.alt.txt
    wildcards: HOME=/home1/zhuyixin/zhuyixin_proj/AssmQuality, species=test
    affected files:
        NA
unlocking
removed all locks
